MmSupervisorPkg/Core/Request/Request.h

```diff
@@ -69,4 +69,21 @@ ProcessVersionInfoRequest (
   OUT MM_SUPERVISOR_VERSION_INFO_BUFFER  *VersionInfoBuffer
   );
 
+/**
+  Function that returns supervisor version information to requesting entity.
+  Calling this function will also block the supervisor memory pages from being updated.
+
+  @param[out] VersionInfoBuffer     Pointer to hold returned version information structure.
+
+  @retval EFI_SUCCESS               The security policy is successfully gathered.
+  @retval EFI_SECURITY_VIOLATION    If VersionInfoBuffer buffer is not pointing to designated supervisor buffer
+  @retval EFI_ACCESS_DENIED         If request occurs before MM foundation is setup.
+  @retval EFI_COMPROMISED_DATA      Supervisor image buffer does not pass minimal PeCoff check.
+
+ **/

```

Comment:
Looks like the function description was copied from `ProcessVersionInfoRequest ()` and not updated.

Reply:
Description updated.